app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,108 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskServiceTest {
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+    private static final String UPDATE_POSTFIX = "!!!";
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+
+        Task task = new Task();
+        task.setTitle(TASK_TITLE);
+        taskService.createTask(task);
+    }
+
+    @Test
+    @DisplayName("TaskService 클래스의  getTasks 메소드는 List<task>를 반환한다")
+    void getTasks() {
+        List<Task> tasks = taskService.getTasks();
+        assertThat(tasks).hasSize(1);
+
+        Task task = taskService.getTasks().get(0);
+        assertThat(task.getId()).isEqualTo(1L);
+        assertThat(task.getTitle()).isEqualTo(TASK_TITLE);
+    }
+
+    @Test
+    @DisplayName("TaskService 클래스의 getTask 메소드는 id가 있다면 해당 Task를 반환한다")
```

Comment:
이 설명문은 아쉬운 점이 있습니다. 다음은 제가 금방 작성한 설명문인데요, 이 설명문과 비교했을 때 어떤 점이 다른가요?
```suggestion
    @DisplayName("TaskService 클래스의 getTask 메소드는 주어진 id에 해당하는 Task가 있다면 해당 Task를 리턴한다")
```